### PR TITLE
Undefined varibale to_path in PackageRuntime.dispatcher (Issue/#288)

### LINKED ISSUE #288: undefined variable in fedn/fedn/common/control/package.py
**Severity**
- [x ] Critical/Blocker (select if the issue makes the application unusable or causes serious data loss)
- [ ] High (select if the issue affects a major feature and there is no workaround or the available workaround is very complex)
- [ ] Medium (select if the issue affects a minor feature or affects a major feature but has an easy enough workaround to not cause any major inconvenience)
- [ ] Low (select if the issue doesn't significantly affect the user experience, like minor visual bugs)

**Describe the bug**
on line 199 a variable: to_path is used without beeing defined, this caused the fedbird project to crash.

**Environment:**
 - OS: [e.g. mac OS]
 - Version: [e.g. macOS Catalina]

**Reproduction Steps**

**Expected behavior**

**Screenshots**

**Contact Details**
An e-mail address in case we need to contact you for further details regarding this issue.

### PR DESCRIPTION
**Pull request by Wrede**
## Status

- [x] Ready
- [ ] Draft
- [ ] Hold

## Description
Fixes #288, also cleaned up unused variables.

## Types of changes

What types of changes does your code introduce to FEDn?

- [ ] Hotfix (fixing a critical bug in production)
- [x] Bugfix
- [ ] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/fedn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included tests to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have added a reviewer for this pull request
- [ ] I have added myself as an author for this pull request

## Further comments

Tested with mnist-keras case, should be tested on a case where fedn.yaml is not default. Suggestion on a case?

### CODE REVIEW

File: fedn/fedn/common/control/package.py [hunk 1]
```diff
@@ -212,19 +212,14 @@ def dispatcher(self, run_path):
...
-        dispatch_dir = self.dir
         from_path = os.path.join(os.getcwd(), 'client')
-        import time
-        dirname = time.strftime("%Y%m%d-%H%M%S")
-
+        
         from distutils.dir_util import copy_tree
         copy_tree(from_path, run_path)
 
-        os.chdir(run_path)
```
Comment (ahellander) [inline fedn/fedn/common/control/package.py:223]: Why can we remove this?

Comment (Wrede) [reply fedn/fedn/common/control/package.py:223]: Is not the question, why is it there? Absolute paths are being used everwhere, even in client.py:
``` 
from fedn.common.control.package import PackageRuntime
pr = PackageRuntime(os.getcwd(), os.getcwd())
``` 

Why is it like this?

**Comment by Wrede** [comment]
Closed in favor of #290

File: fedn/fedn/common/control/package.py (hunk 1 above)
Comment (ahellander) [reply fedn/fedn/common/control/package.py:223]: @morganekmefjord Maybe you remember?

Comment (Wrede) [reply fedn/fedn/common/control/package.py:223]: This PR has been closed due to build failure, please continue review in PR #290

**Pull request closed without merging**